.drone.yml

```diff
@@ -64,6 +64,31 @@ pipeline:
       matrix:
         TEST_SUITE: owncloud-coding-standard
 
+  phpstan:
+    # phpstan requires php7.1
+    image: owncloudci/php:7.1
+    pull: true
+    environment:
+      - COMPOSER_HOME=/var/www/owncloud/apps/testing/.cache/composer
+    commands:
+      - make test-php-phpstan
+    when:
+      matrix:
+        TEST_SUITE: phpstan
+
+  phan:
+    # phan requires a recent php-ast extension
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+      - COMPOSER_HOME=/var/www/owncloud/apps/testing/.cache/composer
+    commands:
+      - make test-php-phan
+    when:
+      matrix:
+        TEST_SUITE: phpunit
+        PHAN: true
```

Comment:
@patrickjahns we can only run ``phan`` if ``PHP_VERSION`` is ``7.*``
I put a boolean ``PHAN`` in here, and only added it to the matrix for 1 phpunit job of PHP 7.0 7.1 and 7.2. Without that, it will try and run for PHP 5.6 also, which will die.

Alternatives would be:
- put a check in the commands section, and only do ``make test-php-phan`` when the PHP version is >=7
- put a check in ``Makefile`` to only really do the ``phan`` for PHP>=7, and emit an information message for PHP<7, so that it will "work" with PHP5.6 but be a NOOP
- or?